docs/website/docs/intro.md

```diff
@@ -4,10 +4,27 @@ sidebar_position: 1
 
 # Introduction
 
+![dlt pacman](/img/dlt-pacman.gif)
+
 ## Summary
 
+`dlt` is an open source library that you include in a Python script to create a highly 
+scalable, easy to maintain data [pipeline](./glossary.md#pipeline) that is straightforward 
+to deploy. Once you set it up, it will then automatically turn JSON returned by any 
+[source](./glossary.md#source) (e.g. an API) into a live dataset stored in the 
+[destination](./glossary.md#destination) of your choice (e.g. Google BigQuery).
+
 ## Who should use `dlt`?
 
+Anyone who solves problems in their job using Python should use `dlt` (e.g. data scientists, 
+data analysts, etc).
+
 ## What does `dlt` do?
 
-## How does `dlt` work?
\ No newline at end of file
+`dlt` is used to automate fetching data for others, copying production data to somewhere else, putting data from an API into a database, getting data for dashboards that automatically refresh with new data, and more.
+
+## How does `dlt` work?
+
+`dlt` extracts data from a [source](./glossary.md#source), normalizes it to a 
```

Comment:
`dlt` extracts data from a source, inspects its structure to create a [schema], normalizes and verifies the data, and then...